modules/mono/editor/GodotTools/GodotTools/Build/MSBuildPanel.cs

```diff
@@ -135,11 +135,13 @@ public override void _Ready()
             _buildMenuBtn = new MenuButton { Text = "Build", Icon = GetThemeIcon("Play", "EditorIcons") };
             toolBarHBox.AddChild(_buildMenuBtn);
 
-            var buildMenu = _buildMenuBtn.GetPopup();
-            buildMenu.AddItem("Build Solution".TTR(), (int)BuildMenuOptions.BuildSolution);
-            buildMenu.AddItem("Rebuild Solution".TTR(), (int)BuildMenuOptions.RebuildSolution);
-            buildMenu.AddItem("Clean Solution".TTR(), (int)BuildMenuOptions.CleanSolution);
-            buildMenu.IdPressed += BuildMenuOptionPressed;
+            var buildMenu = (PopupMenu)_buildMenuBtn.GetPopup();
+            if (buildMenu != null) {
```

Comment:
It shouldn't be null but since I have to cast here, I thought I'd add a check just in case.

Reply:
The cast will throw if the returned value is not cast-able to the requested type. Maybe you wanted to use `as` here? The `as` keyword will return null if it can't be cast-ed instead of throwing. But maybe I'm overthinking this, since we never call `SetPopup` it will never return anything other than the default which is the `PopupMenu` setup in the constructor of `MenuButton`.

Reply:
I just mostly hoped it would work similar to our Object casting in C++, but I don't quite remember C#.

I can change it to whichever makes sense, just tell me what would be the best approach here.

Reply:
Since this will never fail, the cast is fine but the null check seems unnecessary and personally I think we want it throw if it fails rather than failing silently.